server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ClearCachePlan.java

```diff
@@ -35,4 +39,17 @@ public ClearCachePlan() {
   public List<PartialPath> getPaths() {
     return Collections.emptyList();
   }
+
+  @Override
+  public void serialize(DataOutputStream stream) throws IOException {
+    stream.writeByte((byte) PhysicalPlanType.CLEARCACHE.ordinal());
+  }
+
+  @Override
+  public void serialize(ByteBuffer buffer) {
+    buffer.put((byte) PhysicalPlanType.CLEARCACHE.ordinal());
+  }
+
+  @Override
+  public void deserialize(ByteBuffer buffer) throws IllegalPathException {}
```

Comment:
why deserialize do not implementation？ How to deserialize this execution plan?

Reply:
There are no fields to deserialize